public/components/report_definitions/report_trigger/report_trigger.tsx

```diff
@@ -50,9 +50,34 @@ import {
   MONTHLY_DAY_SELECT_OPTIONS,
 } from './report_trigger_constants';
 
-export function ReportTrigger() {
+let trigger_params = {
+  schedule_type: '',
+  schedule: {},
+};
+
+let trigger_schema = {
+  trigger_type: '',
+  trigger_params: {},
+};
+
+let temp_trigger_params = {
+  interval: {
+    period: '7',
+    unit: 'DAYS',
+    start_time: '1553112384',
```

Comment:
what's does this timestamp means? could we make a const?

Reply:
Will extract to const- is a placeholder default values, pending potential update to our data model